.github/workflows/BuildImage.yml

```diff
@@ -7,6 +7,7 @@ env:
   ENDPOINT: "linuxserver/mods" #don't modify
   BASEIMAGE: "radarr" #replace
   MODNAME: "striptracks" #replace
+  MULTI_ARCH: "true" #set to true if needed
```

Comment:
this can be `false` as this mod installs the same files on all arches and wouldn't benefit from separate arch specific versions.

Reply:
Okay.  Done.